.gh-issues/issue-10-stop-recording-button-does-not-work.md

```
@@ -0,0 +1,49 @@
+## Summary
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)